libipl/libipl.H

```diff
@@ -82,6 +89,16 @@ void ipl_log(int loglevel, const char *fmt, ...);
 
 void ipl_set_error_callback_func(ipl_error_callback_func_t fn);
 
+/*
+ * @Brief Process PLAT error to add callout/deconfig and callback
+ * detailsipl_error_type
+ * @param fapiRC - FAPI return code
+ * @param target - target pointer
+ * @param deconfig - true deconfig the target 
+ */
+void ipl_process_fapi_error(const fapi2::ReturnCode& fapirc,
+	struct pdbg_target *target, bool deconfig=true);
```

Comment:
Do we really need `deconfig` parameter? also as a default argument? 
I think we need to decide that based on RC, right? no?

Reply:
in prep for poweroff hwp we do not want to deconfig

Reply:
cannot we decide that based on the error code? I guess each hwp plat error might behave differently, right?